src/util/address.rs

```diff
@@ -76,11 +84,12 @@ pub enum Error {
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match *self {
-            Error::Base58(ref e) => write!(f, "base58: {}", e),
-            Error::Bech32(ref e) => write!(f, "bech32: {}", e),
+            Error::Base58(_) => write!(f, "base58 address encoding error"),
+            Error::Bech32(_) => write!(f, "bech32 address encoding error"),
```

Comment:
`f.push_str("...")` would be a bit more performant but almost certainly doesn't matter. Feel free to ignore.

Reply:
Corrected